### PR TITLE
api: add new endpoint to get the transactions count

### PR DESCRIPTION
**Pull request by mariajdab**
vocdoni/interoperability#31

### CODE REVIEW

**Comment by mariajdab** [comment]
Tried to replicate the error in `job_compose_test` but it was not possible

File: apiclient/blockchain.go [hunk 1]
```diff
@@ -128,3 +129,23 @@ func (c *HTTPclient) OrganizationsBySearchTermPaginated(
...
+		return 0, err
+	}
+	if code != apirest.HTTPstatusOK {
+		return 0, fmt.Errorf("%s: %d (%s)", errCodeNot200, code, resp)
+	}
+	txsCount := new(struct {
+		Count uint64 `json:"count"`
+	})
+
+	err = json.Unmarshal(resp, txsCount)
```
Comment (p4u) [inline apiclient/blockchain.go]: We always try to do (when possible) `if err = ...`

Comment (p4u) [reply apiclient/blockchain.go]: Actually you can do:

`return txsCount.Count, json.Unmarshal(resp, txsCount)`

**Review by p4u — APPROVED** [review]
Great work.
Please, remember to post a message on the interoperability repository to announce the method is already implemented.

**Comment by p4u** [comment]
squash your commits please

**Comment by mariajdab** [comment]
For now, I will not include `censusize end2endtest`. And I will check with @altergui why the `job_compose_test` is failing with: `account not found`, but the docker local test and using the api seems ok

**Comment by p4u** [comment]
I think you should also remove the changes to start_test.sh
The new tests have been introduced by the last PR merged, see https://github.com/vocdoni/vocdoni-node/blob/master/dockerfiles/testsuite/start_test.sh#L105

**Comment by p4u** [comment]
@mariajdab I think we are fine for squash & merge

**Pull request merged**